workflow/compare.rb

```diff
@@ -0,0 +1,86 @@
+require 'csv'
```

Comment:
Can you move this file to workflow/tests/compare.rb?

Reply:
I'll need to also remove this item from the Rakefile.

Reply:
Good point. I'm okay with that.